mgmt-fixes/Makefile

```diff
@@ -0,0 +1,14 @@
+-include ../setup-env.mk
+-include ../helm-cmd.mk
+HELM_CMD ?= helm upgrade --install
+
+deploy:
+	${HELM_CMD} mgmt-fixes \
+		deploy/helm/mgmt-fixes/
+#	kubectl create namespace mgmt-fixes --dry-run=client -o json | kubectl apply -f - && \
+#		--namespace mgmt-fixes
```

Comment:
Is this a leftover from a possible dedicated namespace or do we need it?

Reply:
Yup, obsolete bit I forgot to remove, gone now.